src/core/Calculation.ts

```diff
@@ -127,6 +127,20 @@ export function getPathKeyframeIndexes(path: Path, segmentIndexes: IndexBoundary
   return ikf;
 }
 
+export function simplePoints(path: Path): Point[] {
```

Comment:
I believe you can rename it to a better name. Also, write test cases for your function and add comments to describe how this function works. What is the expected input and output? Functions in the `Calculation.ts` are required to have test cases.